Apps/SampleData/models/BoxInstanced/BoxInstanced.gltf

```diff
@@ -86,20 +86,18 @@
             "TRANSLATION": 3,
             "ROTATION": 4,
             "SCALE": 5,
-            "FEATURE_ID_0": 6
+            "_FEATURE_ID_0": 6
           }
         },
         "EXT_mesh_features": {
-          "propertyTables": [
-            0,
-            1
-          ],
           "featureIds": [
             {
+              "propertyTable": 0,
```

Comment:
ah I'll update the documentation and glTF models.

the `array -> dictionary`... that's best done in its own PR since that's going to mean we have to swap out `featureIdIndex` (Number) for... idk `featureIdSet` (String)

And the removal of implicit feature IDs is best done when I split out `EXT_structural_metadata` so I can replace it with property mappings. (which might get messy given that things are now accessors instead of bufferViews)

I'll open some issues for these.

Reply:
Opened https://github.com/CesiumGS/cesium/issues/10095 for the first one

Reply:
Opened https://github.com/CesiumGS/cesium/issues/10096 for discussing the second one (as there may be some conceptual overlap between this and the implicit tiling part)